### DISCUSSION TITLE
通过rclone连接webdav的时候,上传文件夹可能会创建二个一样的目录

### DISCUSSION THREAD

**Discussion by dfaofeng** [messense/aliyundrive-webdav#152] [General]
![image](https://user-images.githubusercontent.com/14891398/141914655-cd2903a1-87b2-4172-b6b8-fabb4e1b1fca.png)
可能是bug?还有大家用国外的vps晚上8点后上传到网盘可能会时候,提示超时,不知道是不是vps服务商的问题

**Comment by messense** [comment]
一部分是阿里云盘设计问题，它是用 `file_id` 来标识文件/文件夹的，接口层面允许重名。
